package.json

```diff
@@ -62,6 +65,8 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.3",
     "eslint-plugin-react": "^7.19.0",
-    "eslint-plugin-react-hooks": "^3.0.0"
+    "eslint-plugin-react-hooks": "^3.0.0",
+    "prettier": "2.0.4",
+    "prettier-eslint": "^9.0.1"
```

Comment:
This is a bit out of place here.  If we're going to declare prettier, let's not do it in a feature branch.  Can you redo this file and not commit these two lines?

Reply:
Happy to do this. We're keeping all of the ESLint stuff though right? Including 
`"@typescript-eslint/eslint-plugin": "^2.28.0",
"eslint-config-prettier": "^6.10.1",
"eslint-plugin-prettier": "^3.1.3",`?